tests/test_mockgun.py

```diff
@@ -269,6 +269,19 @@ def test_find_by_sub_entity_field(self):
         )
         self.assertEqual(len(items), 1)
 
+    def test_find_with_none(self):
+        """
+        Ensures comparison with multi-entity fields and None works.
+        """
+        items = self._mockgun.find("PipelineConfiguration", [["users", "is", None]], ["users"])
+        self.assertEqual(len(items), 1)
+        self.assertEqual(items[0]["users"], [])
+
+        items = self._mockgun.find("PipelineConfiguration", [["users", "is_not", None]], ["users"])
+        self.assertEqual(len(items), 3)
+        for item in items:
```

Comment:
Minor: As there is not message that details the test, I would go for a single `self.assertTrue(all(len(item("users"]) > 0 for item in items, msg="All the queried items should be have no users.")`

It should speed up the test and increase readability